src/EGraphs/egraph.jl

```diff
@@ -245,7 +244,11 @@ end
 
 function add_class_by_op(g::EGraph, n, eclass_id)
   key = IdKey(v_signature(n))
-  vec = get!(g.classes_by_op, key, Vector{Id}())
+  vec = get(g.classes_by_op, key, nothing)
+  if isnothing(vec)
+    vec = Id[eclass_id]
+    g.classes_by_op[key] = vec
+  end
```

Comment:
Why not `vec = get!(g.classes_by_op, key, Id[eclass_id])`?

Reply:
The idea was that when Id[eclass_id] is supplied as an argument we always create a new vector that we immediately discard when it already exists. 
Probably it would be best to use 
```julia
get!(Vector{Id}, g.classes_by_op, key)
```